src/pocket/PocketSynthetic.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Testing } from 'cdktf';
+import { PocketSyntheticCheck, PocketSyntheticProps } from './PocketSynthetic';
+
+const config: PocketSyntheticProps = {
+  config: {
+    url: 'acme.getpocket.dev',
+    path: '/healthz',
+    port: '443',
+    protocol: 'https:'
+  },
+  environment: 'test',
+  artifactS3Location: 's3://not-a-real-thing'
```

Comment:
should we create a default S3 bucket if a location is not provided?

Reply:
Having a bucket per check seems like a lot to me but I could be convinced otherwise!